jenkinsdocker/buildFromLocal.xml

```diff
@@ -29,41 +29,38 @@
     <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@3894.3896.vca_2c931e7935">
         <script>pipeline {
             environment {
-            STAGE = 'gcs1'
-            }
+                    STAGE = 'gcs1'
```

Comment:
what is this stage name for? (I think I probably added it - it should be something more descriptive)

Reply:
I think this is the stage name that would be used in a deployment of the tools to ECS--updating now